Tests/KSCrashRecordingCoreTests/KSObjC_Tests.m

```diff
@@ -55,7 +55,9 @@ @implementation SomeSubclass
 
 @end
 
-@interface KSObjC_Tests : XCTestCase @end
+// TODO: Fix tagged pointer operations
+//@interface KSObjC_Tests : XCTestCase @end
```

Comment:
I would rather disable it in generated SPM Xcode project. Schemas are commited.

Reply:
Sounds good. Yet this would not work for `swift test`, right? Will double check

Reply:
> Yet this would not work for `swift test`, right? 

Oh, I think it wouldn't

Reply:
We can try to use `xcodebuild test`

Reply:
Let's keep the disabling in code for consistency for now and use `swift test`

Reply:
Does `swift test` have all the necessary environment that Xcode has? Can we expect code tested with `swift test` to behave the same in applications built with Xcode?